contracts/external/traders/Expiry.sol

```diff
@@ -234,6 +265,59 @@ contract Expiry is
 
     // ============ Private Functions ============
 
+    function callFunctionSetExpiry(
+        address sender,
+        bytes memory data
+    )
+        private
+    {
+        (
+            CallFunctionType callType,
+            SetExpiryArg[] memory expiries
+        ) = abi.decode(data, (CallFunctionType, SetExpiryArg[]));
+
+        assert(callType == CallFunctionType.SetExpiry);
+
+        for (uint256 i = 0; i < expiries.length; i++) {
+            SetExpiryArg memory exp = expiries[i];
+            uint32 timeDelta = exp.timeDelta;
+            if (exp.account.owner != sender) {
```

Comment:
this should never really happen... I wonder if we should either `require` this or not check it at all

Reply:
Need users to be able to set expiry on themselves as usual